### DISCUSSION TITLE
Unable to get SW360 to work because of missing users in Couch DB

### DISCUSSION THREAD

**Discussion by MNesche** [eclipse-sw360/sw360#2382] [Q&A]
Hi,
we've set up SW360 v18.0.0 and everything necessary via Docker according to the descriptions.
Couch DB is version 3.2.3.
Unfortunately, we're not able to get SW360 working, because the users are not in Couch DB and we haven't found a way, to get the users to Couch DB.
The databases in SW360 have been created successfully.
Importing users via file doesn't change anything.
I'd be pleased for any help or hint pointing in the right direction to fix this issue.
Please let me know, if you need any further informations.

![image](https://github.com/eclipse-sw360/sw360/assets/153277710/0adc7d53-018e-4da2-825c-084fcef60134)
![image](https://github.com/eclipse-sw360/sw360/assets/153277710/412b9429-0df0-49dd-adaa-d232f5e69f1c)
![image](https://github.com/eclipse-sw360/sw360/assets/153277710/94b08eae-2803-49cc-b96d-8962ff8aa80e)

Thanks a lot in advance.
Kind regards.
Enzo

**Comment by ag4ums** [comment]
@MNesche, did you follow the documentation https://eclipse.dev/sw360/docs/deployment/legacy/deploy-liferay7.4/ while configuring liferay and uploading userdata

**Reply by MNesche** [reply]
Dear Anupam,
thank you for your reply.
Yes, we followed the documentation and I just did reimport the *.lar-files and users.
I did assign department, roles, group to a user but even that didn't change a thing.
However, the problem still exists.

The issue looks similar to https://github.com/eclipse-sw360/sw360/issues/1686
Unfortunately, there's no fix mentioned.

A couple more features get the "... is temporarily unavailable" message:
Projects, Components, Admin/Department, Admin/Schedule Administration

When trying to update a user, the following message appears:
![image](https://github.com/eclipse-sw360/sw360/assets/153277710/8dafdbe8-a452-4244-b003-dbce6c3d6936)

Thanks a lot in advance for your time and effort in helping to resolve the issue.
